go-fuzz-build/main.go

```diff
@@ -32,6 +33,14 @@ var (
 	GOROOT  string
 )
 
+func makeTags() string {
+	tags := "gofuzz"
+	if len(*flagTag) > 0 {
+		tags += " " + *flagTag
+	}
+	return tags
```

Comment:
Joined tags should have form `-tags="tagA,tagB"` try it out for yourself.

See also: https://github.com/golang/go/issues/8772#issuecomment-253946071

Thins get a bit tricky when you have multiple tags in play.